src/schema/v2/__tests__/salesConnection.test.ts

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable promise/always-return */
+import { runQuery } from "schema/v2/test/utils"
+import gql from "lib/gql"
+import sinon from "sinon"
+
+describe("artworksConnection", () => {
```

Comment:
Were you intending to also test the `registered` flag here?

Reply:
Ah yes I was. Also this was a bad copy-&-paste, I'll fix that as well.